### DISCUSSION TITLE
On-Demand File Access

### DISCUSSION THREAD

**Discussion by zaxlofful** [abraunegg/onedrive#2514] [General]
I know this is one of the features that is listed as "missing" that the normal OneDrive client achieves.

Is there a roadmap for this feature at all?

**Comment by abraunegg** [comment]
@ZaxLofful 
Not at this stage.

Please refer to: https://github.com/abraunegg/onedrive/issues/757

**Comment by zaxlofful** [comment]
Yes, I read that thread and it doesn't answer the question about the future plans.

OneDriver is a very crude workaround, that only does half of the implementation of the feature in question.

**Comment by abraunegg** [comment]
@ZaxLofful 
> Is there a roadmap for this feature at all?

As per above (Not at this stage.) - there is no plan or roadmap to develop this feature at present.

**Comment by zaxlofful** [comment]
That is unfortunate

**Reply by abraunegg** [reply]
Unsure why this is unfortunate - given all the development effort with v2.5.x at this stage, which will resolve the following bugs & feature requests:
* https://github.com/abraunegg/onedrive/issues/232
* https://github.com/abraunegg/onedrive/issues/1251
* https://github.com/abraunegg/onedrive/issues/2359
* https://github.com/abraunegg/onedrive/issues/2434
* https://github.com/abraunegg/onedrive/issues/2475
* https://github.com/abraunegg/onedrive/issues/2501

and will also lay the groundwork for:
* https://github.com/abraunegg/onedrive/issues/779
* https://github.com/abraunegg/onedrive/issues/1300
* https://github.com/abraunegg/onedrive/issues/2360

There is only so much free time in a day | week | month to work on aspects .....
